packages/components/utils/src/MissingContent/MissingContent.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: 'MissingContent'
+type: 'component'
+slug: /utils/missingcontent/
+section: 'utils'
```

Comment:
I don't think we should put components rendered to the user in utils, but if you're thinking there's no other obvious place to put it I would suggest something like an `empty-state` directory that we can expand with other types of empty state components in the future.